source/javascripts/services/_appService.js.erb

```diff
@@ -471,6 +496,29 @@
 				appService.defaultMachineType = appService.savedDefaultMachineType;
 			};
 
+			appService.getDefaultLicensePoolId = function() {
+				appService.defaultLicensePoolId =
+					appService.appConfig &&
+					appService.appConfig.meta &&
+					appService.appConfig.meta["<%= data[:constants][:meta_bundle_ids][:bitrise_io] %>"] &&
+					appService.appConfig.meta["<%= data[:constants][:meta_bundle_ids][:bitrise_io] %>"].license_pool_id;
+				if (!appService.defaultLicensePoolId) {
+					return $q.reject(new Error("<%= data[:strings][:machine_type][:invalid_set_in_bitrise_yml_meta] %>"));
```

Comment:
Is it indeed an error if a license pool ID is not set?

Reply:
removed all default license pool related changes